Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Person.hs

```diff
@@ -73,6 +73,24 @@ data CreatePersonReq = CreatePersonReq
   }
   deriving (Generic, ToJSON, FromJSON, ToSchema)
 
+data ChangeEmailByAdminReq = ChangeEmailByAdminReq
```

Comment:
can you add request validation?

Reply:
just take "email" as input. confirm email should be at UI.